src/jira_client.ts

```diff
@@ -101,19 +104,24 @@ export interface IJiraAPIClient {
 }
 
 export function JiraAPIClient(
-  baseUrl: string,
-  jiraEmail: string,
-  token: string,
-  boardId: string,
-  statusCompleteLabel: NonEmptyString,
+  baseUrl: NonEmptyString,
+  config: {
+    readonly boardId: NonEmptyString;
+    readonly email_tag_prefix: NonEmptyString;
+    readonly ente_tag_prefix: NonEmptyString;
+    readonly jiraEmail: NonEmptyString;
+    readonly service_tag_prefix: NonEmptyString;
+    readonly status_complete: NonEmptyString;
```

Comment:
We should use always `camelCase` names for local variables. We use `snaked_case` only for properties exported through API interface.

Reply:
Fix with f922cc5